drivers/fuel_gauge/sbs_gauge/sbs_gauge.h

```diff
@@ -47,8 +47,33 @@
 
 #define SBS_GAUGE_DELAY                     1000
 
+/*
+ * Nearly all cutoff payloads are actually a singular value that must be written twice to the fuel
+ * gauge. For the case where it's a singular value that must only be written to the fuel gauge only
+ * once, retransmitting the duplicate write has no significant negative consequences.
```

Comment:
Delete reference to retransmitting duplicate value, since your code doesn't require that.